src/redux/books/Books.js

```diff
@@ -13,13 +16,34 @@ export const removeBook = (payload) => ({
   payload,
 });
 
+export const loadBook = (payload) => ({
+  type: GET_BOOK,
+  payload,
+});
+
+export const loadBooks = () => (dispatch) => {
+  getBooks().then((books) => {
+    Object.keys(books).forEach((ID) => {
+      const booksId = {};
+      booksId.id = ID;
+      booksId.title = books[ID][0].title;
+      booksId.category = books[ID][0].category;
+      dispatch(loadBook(booksId));
+    });
+  });
+};
```

Comment:
A good example of creating an asynchronous thunk!

Reply:
Thanks, @D4nielJ 

Thunk is already used in store

![image](https://user-images.githubusercontent.com/68148157/146252523-422d8f9b-e2df-45f8-a389-0e100c9af80d.png)

Reply:
Hello Emmy, to clarify: Thunk is middleware that allows the store to dispatch Asynchronous Functions named thunks. I'm not asking you to install the plugin again, it is correctly installed. I'm asking you to remove both async functions inside of the reducer to thunks. [I hope this is more clear and if you need to read more about thunks you can do it here.](https://redux.js.org/usage/writing-logic-thunks)

Reply:
Asynchronous is implemented in API 

![image](https://user-images.githubusercontent.com/68148157/146256308-28e384ad-5c0b-424d-b1dd-5a6532547fe1.png)